styles/globals.css

```diff
@@ -1,26 +1,268 @@
-html,
-body {
-  padding: 0;
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+p,
+a {
+  font-family: "Inter", sans-serif;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: "Rozha One", "serif";
+}
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active {
+  @apply bg-secondary-300;
+  -webkit-box-shadow: 0 0 0 30px #f5f6f8 inset !important;
+}
+
+.is-sticky {
+  @apply bg-secondary-25;
+}
+p.paragraphs::first-letter {
+  @apply text-secondary-600;
+
+  font-size: 330%;
+  line-height: 2rem;
+  padding: 6px 1px;
+  margin-right: 2px;
+  float: left;
+}
+
+.scrollbar::-webkit-scrollbar {
+  @apply hidden;
+  width: 0;
+  background: transparent; /* make scrollbar transparent */
+}
+
+/* make scrollbar transparent on Firefox, IE and Edge*/
+.scrollbar {
+  @apply scroll-p-0;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
+.form-radio,
+.form-checkbox {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  -webkit-print-color-adjust: exact;
+  color-adjust: exact;
+  display: inline-block;
+  vertical-align: middle;
+  background-origin: border-box;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  flex-shrink: 0;
+  color: #f2fbff;
+
+  border-color: #8d94a0;
+  border-width: 1px;
+}
+.form-checkbox {
+  background-color: #fff;
+}
+
+.form-radio {
```

Comment:
It might be a better option to add some of these CSS. classes into a different file such that we have form.css, rs-picker.css to ensure that the file is more manageable.